app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -174,8 +192,8 @@
             android:text="@string/storyline"
             android:textColor="@color/storyline"
             android:textSize="14sp"
-            app:layout_constraintStart_toEndOf="@+id/guideline_left"
             app:layout_constraintEnd_toStartOf="@id/guideline_right"
+            app:layout_constraintStart_toEndOf="@+id/guideline_left"
             app:layout_constraintTop_toBottomOf="@+id/storyline_head" />
 
```

Comment:
android:id="@+id/storyline_head"

android:layout_width="wrap_content"
app:layout_constraintStart_toEndOf="@+id/guideline_left"

what about right border for long text?


please play with long reviews text/bit reviews textSize
app:layout_constraintTop_toBottomOf="@+id/star1"